insights/specs/default.py

```diff
@@ -621,6 +621,7 @@ def pmlog_summary_file(broker):
     satellite_custom_ca_chain = simple_command(
         '/usr/bin/awk \'BEGIN { pipe="openssl x509 -noout -subject -enddate"} /^-+BEGIN CERT/,/^-+END CERT/ { print | pipe } /^-+END CERT/ { close(pipe); printf("\\n")}\' /etc/pki/katello/certs/katello-server-ca.crt',
     )
+    satellite_custom_hiera = simple_file("/etc/foreman-installer/custom-hiera.yaml")
```

Comment:
Is this line has something with this PR? Otherwise please modify it in another PR. Thanks.

Reply:
@xiangce, no relationship, I just found it's disordered when I add this one, sure, I'll update it in another PR.